Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -323,3 +355,32 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 	}
 	return utilerrors.NewAggregate(allErrs)
 }
+
+// mustPrintWithKinds determines if printer output
+// should print multiple kinds of objects, even if
+// "all-kinds" flag is not set
```

Comment:
The flag name is "show-kind".
Anyway, this function has nothing to do with the flag. It doesn't need to know that flag exist nor anything.

While it's tempting to mention it while it's hot in our minds, long term it's a bad idea. For example, imagine we rename the flag to "prefix-with-kind", or anything ("all-kind"!) -- this comment will become rotten because nobody will come look at this function when changing the flag.

It doesn't help understand what the function does, please let's get rid of it :)